democracy/tests/test_comment.py

```diff
@@ -55,7 +55,7 @@ def lookup_field(request):
 def test_56_add_comment_to_section_without_authentication(api_client, default_hearing, get_comments_url_and_data):
     section = default_hearing.sections.first()
     url, data = get_comments_url_and_data(default_hearing, section)
-    response = api_client.post(url, data=data)
+    response = api_client.post(url, data=data, format='json')
```

Comment:
Apparently there seems to be a global setting for this :) http://www.django-rest-framework.org/api-guide/testing/#setting-the-default-format